integration/run_test.go

```diff
@@ -38,14 +38,20 @@ import (
 	"k8s.io/client-go/tools/clientcmd/api"
 )
 
-var gkeZone = flag.String("gke-zone", "us-central1-a", "gke zone")
-var gkeClusterName = flag.String("gke-cluster-name", "cluster-3", "name of the integration test cluster")
-var gcpProject = flag.String("gcp-project", "kritis-int-test", "the gcp project where the integration test cluster lives")
-var remote = flag.Bool("remote", true, "if true, run tests on a remote GKE cluster")
-
-var client kubernetes.Interface
+const (
+	kritisPreinstall  = "kritis-preinstall"
+	kritisPostinstall = "kritis-postinstall"
+)
 
-var context *api.Context
+var (
+	gkeZone        = flag.String("gke-zone", "us-central1-a", "gke zone")
+	gkeClusterName = flag.String("gke-cluster-name", "kritis", "name of the integration test cluster")
+	gcpProject     = flag.String("gcp-project", "priya-wadhwa", "the gcp project where the integration test cluster lives")
```

Comment:
The default value here seems unusual. Was this unintentional?

Reply:
Whoops forgot to change that back, thanks :sweat_smile: